anago

```diff
@@ -165,12 +165,12 @@ common::cleanexit () {
 ###############################################################################
 # Simple ACL check to limit nomock runs to a short list of release folks
 check_acls () {
-  case "$USER" in
-    $ACL_LIST) ;;
-            *) logecho "Live releases restricted to certain users!"
-               return 1
-               ;;
-  esac
+  if echo "$ACL_LIST" | tr "|" "\n" | grep -q "$USER"; then
```

Comment:
Ooops, yes, that is broken.  Thanks for the PR.

How about less piping/moving parts with:
```
if echo "$USER" | egrep -w "$ACL_LIST"; then
```